doc/md_src_files/05_07_mission_impact.md

```diff
@@ -7,6 +7,21 @@ When reviewing the list of organizational functions, an organization must first
 
 As mission essential functions are most clearly defined for government agencies, stakeholders in other sectors may be familiar with different terms of art from continuity planning. For example, infrastructure providers in the US may better align with [National Critical Functions](https://www.cisa.gov/national-critical-functions). Private sector businesses may better align with [operational and financial impacts](https://www.ready.gov/sites/default/files/2020-03/business-impact-analysis-worksheet.pdf) in a [business continuity plan](https://www.ready.gov/business-continuity-plan). While the processes, terminology, and audience for these different frameworks differ, they all can provide a sense of the criticality of an asset or assets within the scope of the stakeholder conducting the cyber vulnerability prioritization with SSVC. In that sense they all function quite similarly within SSVC. Organizations should use whatever is most appropriate for their stakeholder context, with Mission Essential Function analysis serving as a fully worked example in the SSVC documents. 
 
+The DoD's concept of Mission Essential Functions (MEF), as referenced in DOD DIRECTIVE 3020.26 DOD CONTINUITY POLICY, is a slightly different use case. Select functions directly related to accomplishing the Department’s mission. Failure to
+perform or sustain these functions, which directly support PMEF, would significantly affect the
+Department of Defense’s ability to provide vital services or exercise authority, direction, and
+control.
+
+To evaluate performance of DoD MEFs, validated by the National Continuity Coordinator, which must be
+performed in order to support the performance of the national essential functions before, during,
+and in the aftermath of an emergency. PMEFs need to be performed continuously or resumed
+within 12 hours after an event and maintained for up to 30 days or until normal operations can be
+resumed.
+
+The DoD's concept of national essential functions is more aligned with FEMA's notion. The select functions that are necessary to lead and sustain the
```

Comment:
So I started off working on adding a reference but in the process I went and looked at the FCD-2 doc to see why DoD and FEMA's perspectives were different -- especially since the concept of a National Essential Function seems like it'd be coming from outside DoD, and what I found makes me question whether we need this change at all. 

If you look at [FCD-2](https://www.fema.gov/sites/default/files/2020-07/Federal_Continuity_Directive-2_June132017.pdf) which is what we're referencing for the MEF definition, its framing seems to match the NEF -> PMEF -> MEF hierarchy that DoD outlines in 3020.26.

From FCD-2's _Purpose_:
>FCD-2 outlines requirements and
provides checklists and resources to assist D/As in identifying and assessing their essential
functions through a risk-based process and in identifying candidate PMEFs that support the
National Essential Functions (NEFs). This FCD provides guidance for conducting a BPA and
BIA to identify essential function relationships, dependencies, time sensitivities, threats,
vulnerabilities, consequences, and mitigation strategies related to the performance of the MEFs
and PMEFs. This FCD also provides direction on the formalized process for submitting D/As’
candidate PMEFs in support of the NEFs.

Which means that the assertion here that DoD's NEF matches FEMA's MEF is inaccurate. I think we need to reframe this either into a smaller change or just leave the doc as-is.

We seemed to be okay previously with just mentioning MEFs without situating them as the bottom of a hierarchy leading to NEFs, which I think makes sense because many organizations outside the USG could use the MEF concept even if they're not specifically required to. 

So I'm wondering if we actually just need to note here that DoD's MEF and FEMA's MEF are drawn from the same definitions, and indicate that they are part of a hierarchy of essential functions without going into much detail to avoid overfitting SSVC's concept (which is _borrowing_ from FEMA anyway) onto how USG defines things.

Reply:
Resolved by commit 791dcabd716c2e681215493b26cba79f3863887b